### DISCUSSION TITLE
[Uno.sdk] The app doesnt build when you add some packages

### DISCUSSION THREAD

**Discussion by Marc-Antoine-Soucy** [unoplatform/uno#16966] [Q&A] [answered]
### Current behavior

When I add package that have a uno dependency in the `Directory.Package.props`, I think, 
like:
`<PackageReference Include="Chinook.BackButtonManager.Uno.WinUI" Version="2.1.0" />`
for example the basic app doesnt build
and I get the error:
`1>C:\p\ExampleUnoTemplate\ExampleUnoTemplate\ExampleUnoTemplate.csproj : error NU1008: Projects that use central package version management should not define the version on the PackageReference items but on the PackageVersion items: Chinook.BackButtonManager.Uno.WinUI;Chinook.BackButtonManager.Uno.WinUI;Chinook.BackButtonManager.Uno.WinUI;Chinook.BackButtonManager.Uno.WinUI;Chinook.BackButtonManager.Uno.WinUI;Chinook.BackButtonManager.Uno.WinUI.
1>Done building project "ExampleUnoTemplate.csproj" -- FAILED.`

### Expected behavior

It should build and I should not get an error. Or maybe some guidance on how to add package that might be platform dependent?

### How to reproduce it (as minimally and precisely as possible)

[ExampleUnoTemplate.zip](https://github.com/user-attachments/files/15517524/ExampleUnoTemplate.zip)
Create an app

What I used: Blank preset
- material theme
- feature toolkit
- rest is default

add 
`<PackageReference Include="Chinook.BackButtonManager.Uno.WinUI" Version="2.1.0" />`
to Directory.Package.props

try to build

### Works on UWP/WinUI

No

### Environment

Uno.WinUI / Uno.WinUI.WebAssembly / Uno.WinUI.Skia

### Affected platforms

WebAssembly, Android, iOS, Windows (WinAppSDK), Build tasks

### IDE

Visual Studio 2022

### IDE version

17.9.6

**Comment by jeromelaban** [comment]
Thanks for the question. [Central Package Management](https://learn.microsoft.com/en-us/nuget/consume-packages/Central-Package-Management) works with `PackageVersion` in `Directory.Packages.props` and `PackageReference` without a version anywhere else.

**Reply by Marc-Antoine-Soucy** [reply]
Yes, and, that's what I did

**Reply by Marc-Antoine-Soucy** [reply]
I created a local git repo right after creating the project and this is the only change I made to the project: 
![image](https://github.com/unoplatform/uno/assets/90481654/e0995c9f-4493-48cd-8d9a-e48db23f2528)

**Reply by Marc-Antoine-Soucy** [reply]
adding the nuget  reference afterwards (without the version) in the cs proj doesnt help either

**Reply by jeromelaban** [reply, marked as answer]
No, you're adding a `PackageReference`, not a `PackageVersion`. Make sure to add the latter, that is the only part that `Directory.Package.props` supports.

**Reply by Marc-Antoine-Soucy** [reply]
thanks

**Reply by agneszitte** [reply]
to recap @Marc-Antoine-Soucy at the end you should have:
In `NameOfYourProject.csproj`:
```
  <ItemGroup>
    <PackageReference Include="Chinook.BackButtonManager.Uno.WinUI" />
  </ItemGroup>
```
And in `Directory.Packages.props`:
```
  <ItemGroup>
    <PackageVersion Include="Chinook.BackButtonManager.Uno.WinUI" Version="2.1.0" />
  </ItemGroup>
```

(**Manage Nuget Packages for Solution...** in Visual Studio add the above correctly normally when adding a new package)
